websites/es/README.md

```diff
@@ -25,7 +25,7 @@ Este proyecto está patrocinado por:
 
 <a href="https://bringyourownideas.com" target="_blank" rel="noopener noreferrer"><img src="https://bringyourownideas.com/images/byoi-logo.jpg" height="100px"></a>
 
-¿Quieres patrocinar este proyecto? [Póngase en contacto conmigo](https://peterthaleikis.com/contact).
+¿Quieres patrocinar este proyecto? [Escríbeme](https://peterthaleikis.com/contact).
```

Comment:
"Escríbeme" or "Escríbe me" @walirazzaq ?

Reply:
I'll check it again. I think Deepl might have combined it 🤦🏼I'll try to cross verify it on some other translation services as well 👍🏼